src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -37,11 +36,11 @@ const DictionaryDetailCard = (dictionary) => {
         <fieldset>
           <legend>Concepts (HEAD version)</legend>
           <p className="paragraph">Total Concepts: { active_concepts }</p>
-          <p className="points">Custom Concepts: 5</p>
-          <p className="points">From CIEL: 1229</p>
+          <p className="points">Custom Concepts: {dictionary.cielConcepts}</p>
```

Comment:
Why have you assigned Custom Concepts the value for Ciel concepts and vice versa?

Reply:
Thanks for noticing those errors. I have implemented your feedback and updated the PR.